manifests/init.pp

```diff
@@ -136,6 +140,7 @@
     proxy_port             => $proxy_port,
     proxy_username         => $proxy_username,
     proxy_password         => $proxy_password,
+    yum_max_speed          => $pulp_max_speed,
```

Comment:
@sean797 it looks like tests are still failing with this switched around. I tried using a $ in fron of yum_max_speed and still tests fail.